set.mm

```diff
@@ -420312,6 +420323,35 @@ the theorem forms ("i", "d" or "g") should be the first of multiple
        </p>
 
        <ul>
+       <li><b>Contributors.</b>
+       The (original) contributor of a theorem is the first one who proved the
+       theorem and added it to set.mm. She/he is mentioned in the "Contributed
+       by" parenthetical within the comment of the theorem, see Metamath Book,
+       p. 142.
+       An exception should be made if a theorem is an (essential) extract or a
+       variant of an already existing theorem.  An example of the latter is
+       the provision of a theorem in deduction form as variant of a theorem in
+       inference form, e.g., ~ mapsn and ~ mapsnd .  In these cases, the
+       contributor of the former theorem (whose proof can often be shortened by
+       the new theorem) should also be the contributor of the new theorem.  The
+       one who provided the new theorem should be mentioned in hte "Revised by"
+       parenthetical (see Metamath Book, p. 143) within the comment of the new
+       theorem.
+       </li>
+
+       <li><b>Usage of Parentheticals.</b>
```

Comment:
This means you give up on the idea of a proof / theorem history overview.  I never knew how important this was to others, that's why I always added a new credits tag, even if my name was already there.

Reply:
I've never been a fan of trying to put an extensive history here. It is both overcomplete, in the sense that it gives a lot of detail which won't be relevant to most readers, and undercomplete, in the sense that a string of parentheticals gives little sense for what changed each time. I would say that a tool like git is better suited to keeping track of all the ways that something changed.